Makefile

```diff
@@ -24,7 +24,7 @@ lint:
 	find -name '*.sh' | grep -v node_modules | xargs shellcheck
 
 validate: generate-examples
-	java -jar bin/org.hl7.fhir.validator.jar build/examples/**/*application_fhir+json*.json -version 4.0.1 -tx n/a | tee /tmp/validation.txt
+	java -jar bin/org.hl7.fhir.validator.jar build/examples/**/*application_fhir+json*.json -version 4.0.1 -tx n/a -extension any | tee /tmp/validation.txt
```

Comment:
It seems like the new version of the validator has changed the default behaviour for validating extensions to FHIR

Without this new parameter we get a lot of validation errors. With the parameter we get the errors as notes which looks to be the same as the output in the old builds